bodo/pandas/_plan.cpp

```diff
@@ -533,9 +558,7 @@ std::vector<int> get_projection_pushed_down_columns(
  *
  */
 static void RunFunction(duckdb::DataChunk &args, duckdb::ExpressionState &state,
-                        duckdb::Vector &result) {
-    throw std::runtime_error("Cannot run Bodo UDFs during optimization.");
-}
+                        duckdb::Vector &result) {}
```

Comment:
Why is this removed? I think the exception was necessary to avoid query planning time data access as the comment suggests but don't remember the exact details.

Reply:
Well, if anything is going to access the data during query planning time it would be this function but if you don't do anything in this function then I am assuming that it won't happen downstream.  I removed it because one of the queries calls this function and so errors out.  I think if you don't do anything in this function you get something like NA that doesn't match anything and so won't trigger any erroneous optimizations.  My investigation of this was somewhat limited so there could be undesired downstream effects.